test/type/index.test-d.ts

```diff
@@ -0,0 +1,9 @@
+import { expectType } from 'tsd';
+import * as sdk from '../../src/types/index';
+import { Client } from '../../src/types/client';
```

Comment:
This should work the same using: `import { Client } from '../../src/types/index';`  I think this would be a slightly better way to test "index". 

`import * as sdk from '../../src/types/index';` tests that all named exports can be combined in a single object and `import { Client } from '../../src/types';` tests that the individual named export works as expected